### DISCUSSION TITLE
runTimer is being autocompleted wrong, I think

### DISCUSSION THREAD

**Discussion by qq735-personal** [Snirozu/Funkin-Script-AutoComplete#5] [General]
![image](https://github.com/Snirozu/Funkin-Script-AutoComplete/assets/49921877/a2ba028b-f78b-40da-ae52-707e93d9e421)
![Screenshot 2023-07-28 092534](https://github.com/Snirozu/Funkin-Script-AutoComplete/assets/49921877/6df03567-4d7d-436c-b1c3-86606eb6559d)

**Comment by Snirozu** [comment]
sorry but what is this abomination!?
you can't have functions inside functions

**Reply by qq735-personal** [reply]
hell if i knew that

**Reply by Snirozu** [reply]
what are you even trying to do man

**Reply by qq735-personal** [reply]
im trying to make a note that makes the window float offscreen temporarily if pressed
im very sorry for the inconvenience and im fixing the problems now
again, sorry for the sass.

**Reply by Snirozu** [reply]
try using this code, haven't tested it but it should work

``` lua
---
--- @param membersIndex int
--- @param noteData int
--- @param noteType string
--- @param isSustainNote bool
---
function goodNoteHit(membersIndex, noteData, noteType, isSustainNote)
    if noteType == 'WindowNote' then
        runHaxeCode("FlxTween.tween(Application.current.window, {x: 320, y: 1080}, 0.9, {ease: FlxEase.CubeIn})")
        runTimer("WindowNote", 2, 1)
    end
end

---
--- @param tag string
--- @param loops integer
--- @param loopsLeft integer
---
function onTimerCompleted(tag, loops, loopsLeft)
    if tag == 'WindowNote' then
        runHaxeCode("FlxTween.tween(Application.current.window, {x: 320, y: 180}, 0.9, {ease: FlxEase.CubeOut})")
    end
end
```

[please learn basics of lua before jumping into psych engine mods!](https://www.lua.org/pil/1.html)

**Reply by qq735-personal** [reply]
thank you! ill bookmark that page for later.
